composer.json

```
@@ -119,6 +119,7 @@
         "psr/simple-cache-implementation": "1.0|2.0|3.0"
     },
     "conflict": {
+        "carbonphp/carbon-doctrine-types": "^3.0",
```

Wondering if the below is better @timacdonald?
```suggestion
        "carbonphp/carbon-doctrine-types": ">=3.0",
```